renovate.json

```diff
@@ -4343,6 +4352,25 @@
         "Feature:Console"
       ],
       "enabled": true
+    },
+    {
+      "groupName": "node-libs-browser",
+      "matchDepNames": [
+        "node-libs-browser"
```

Comment:
This was last published 6 years ago, and the repository is archived.  Is there an alternative we can use?

Reply:
I exposed this through an internal package and the plan is that we can go over each node lib browser dependency and potentially experiment with a replacement for a newer version. I avoid doing this as part of this PR to exclude complexity and to discard booms to the bundle sizes etc so I just adopted for now the same set webpack v4 was using for the polyfills

Reply:
This is an alternative I've found: https://github.com/niksy/node-stdlib-browser

Reply:
during the development I tried some new packagings of those libs but they bloated the final bundles and thats why I thought it was a better idea to try to upgrade those one by one. I can't remember if I tried that one so will give it a shot 👍

Reply:
update: I tried to use this one locally but I failed to make it because of the ESM/CJS interop. I believe we would have to go with the previous approach I mentioned